Sources/StreamChat/APIClient/Endpoints/MessageEndpoints.swift

```diff
@@ -126,3 +136,39 @@ extension Endpoint {
         )
     }
 }
+
+public struct MessageReactionsQueryFilter: Encodable {
+    enum CodingKeys: String, CodingKey {
+        case type
+        case authorId = "user_id"
+    }
+
+    public var type: MessageReactionType?
+    public var authorId: UserId?
+
+    public init(type: MessageReactionType? = nil, authorId: UserId? = nil) {
+        self.type = type
+        self.authorId = authorId
+    }
+}
```

Comment:
The backend also accepts a mongo db query, but I think in this case this is enough? Because really the only supported queries are `and` and `$eq`. So using our existing query filters for these seems to be an overkill